doc-content/kogito-docs/src/main/asciidoc/drl/proc_drl-rules-create.adoc

```diff
@@ -156,8 +156,8 @@ end
 rule "Underage"
   salience 15
   when
-    $application : /loanApplication
-    /applicant[ age < 21 ]
+    /applicant[ applicantName : name, age < 21 ]
+    $application : /loanApplication[ applicant == applicantName ]
```

Comment:
I rewrote this rule to add a join condition, otherwise there won't be any relationship between the applicant and the loan application and then no way to say that the second refers to the first.

Reply:
Done. Fixed everywhere.